### PR TITLE
Add org admin info

### PR DESCRIPTION
**Pull request by tarzzz**
Addresses #8

### CODE REVIEW

File: broadgauge/webapp.py [hunk 1]
```diff
@@ -188,9 +188,12 @@ def GET(self):
 class org_view:
     def GET(self, id):
         org = Organization.find(id=id)
+        admin = User.find(id = org.admin_id)
+        print admin.name
         if not org:
             raise web.notfound()
-        return render_template("orgs/view.html", org=org)
+   
+        return render_template("orgs/view.html", org=org,admin=admin)
```
Comment (anandology) [inline broadgauge/webapp.py]: Please do this by adding `get_admin` method to Organization model and using org.get_admin() in the template.

**Comment by tarzzz** [comment]
@anandology I have addressed your comments

File: broadgauge/models.py [hunk 2]
```diff
@@ -86,4 +86,10 @@ class Organization(Model):
...
         id = get_db().insert("organization", name=name, city=city, admin_id=admin_user.id, admin_role=role)
-        return cls.find(id=id)
\ No newline at end of file
+        return cls.find(id=id)
+
+    @classmethod
+    def get_admin(cls, id):
+        admin_id = cls.find(id=id).admin_id
+        return User.find(id=admin_id)
+
```
Comment (anandology) [inline broadgauge/models.py:93]: This should be a regular method, not classmethod. 

```
def get_admin(self):
    return User.find(id=self.admin_id)
```

**Comment by anandology** [comment]
Merged in 3f35916019bb47656de9202e7547fe9703e7a966. Thanks @tarzzz .

**Pull request closed without merging**